Python-SMS-Quickstart.py

```diff
@@ -1,35 +1,34 @@
 from __future__ import print_function
-import time
 import freeclimb
 from freeclimb.api import default_api
+from freeclimb.model.message_request import MessageRequest
 import os
 import json
 from flask import Flask, request
 
 configuration = freeclimb.Configuration(
-    # Defining host is optional and default to https://www.freeclimb.com/apiserver
-    host     = "https://www.freeclimb.com/apiserver",
-    # Configure HTTP basic authorization: fc
+    host = "https://www.freeclimb.com/apiserver",
```

Comment:
since this is optional and external users most likely won't need to use it, can we remove?